Terminal.Gui/Views/ColorPicker.cs

```diff
@@ -63,7 +63,8 @@ public void ApplyStyleChanges ()
                     Y = y,
                     Width = textFieldWidth
                 };
-                tfValue.HasFocusChanged += UpdateSingleBarValueFromTextField;
+                tfValue.HasFocusChanged += (s,_)=> UpdateSingleBarValueFromTextField(s);
```

Comment:
I think you should check `!e.NewValue`. You only want to update the values on "leave", not "enter".

Reply:
Fixed in ff6fbc9d80f275032a2f2cdca000002081c53d48